content/docs/user-guide/how-to/share-a-dvc-cache.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'How to Share a Cache Among Projects'
```

Comment:
do we remove the use case file as part of this PR?

Reply:
Not doing that yet. It's basically the same content, yes so we could. But I thought there were enough changes in this PR. Up to you.

Reply:
(being removed in #2587 anyway)

Reply:
I'd err on the side of keeping it until the new use case is ready though. Also removing it may require a redirect (yet another file to change).

Reply:
Okay, got it.